index.js

```diff
@@ -7,42 +9,31 @@ const checkArgs = (from, to, step) => {
   if (!isNumber(from) || !isNumber(to) || !isNumber(step)) {
     throw new Error('Arguments must be numbers');
   }
+  if (from > to && step > 0) {
+    throw new Error('Descending range must have negative step. Got +' + step);
+  }
 };
 
+const isProd = process.env.NODE_ENV === 'production'
```

Comment:
this should allow dropping the checkArgs in production mode when bundling this library for usage on the web